base/system/winlogon/lang/en-US.rc

```diff
@@ -20,6 +20,17 @@ BEGIN
     DEFPUSHBUTTON "&Restart", 1, 80, 91, 68, 14
 END
 
+IDD_SYSSHUTDOWN DIALOGEX 50, 50, 162, 100
+STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_VISIBLE | WS_CAPTION | WS_POPUP
+CAPTION "System Shutdown"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    ICON IDI_WINLOGON, IDC_SYSSHUTDOWNICON, 6, 7, 18, 20
+    LTEXT "Please save everything. All unsaved work will be lost when the system shuts down in:", IDC_SYSSHUTDOWNMSG, 38, 7, 124, 31
```

Comment:
Later on we can think about turning this message a little bit more "professional" like:
`A system shutdown has been initiated by DOMAIN\USER. Please save all your work and terminate your session.`, and then the message can continue with the other sentence you wrote :)
(this would however require to find the DOMAIN and USER name of the person who triggered the shutdown :) )

Reply:
I could just say "A system shutdown has been initiated.", and later on we can add the DOMAIN\USER functionality.

Reply:
Unfortunately (unless there's a bash utility), I won't be able to remove the BOM.

Reply:
I added the more professional message :D